packages/theme_tailor/example/lib/theme_wih_enums.dart

```diff
@@ -1,26 +1,26 @@
-import 'package:flutter/material.dart';
-import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
+// import 'package:flutter/material.dart';
+// import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
 
-part 'theme_wih_enums.tailor.dart';
+// part 'theme_wih_enums.tailor.dart';
 
-enum SuperThemeEnum {
-  light,
-  superLight,
-  dark,
-  superDark,
-}
+// enum SuperThemeEnum {
+//   light,
+//   superLight,
+//   dark,
+//   superDark,
+// }
 
-const themes = ['light', 'superLight', 'dark', 'superDark'];
-const themesEnums = [SuperThemeEnum.light, SuperThemeEnum.superLight, SuperThemeEnum.dark, SuperThemeEnum.superDark];
+// const themes = ['light', 'superLight', 'dark', 'superDark'];
+// const themesEnums = [SuperThemeEnum.light, SuperThemeEnum.superLight, SuperThemeEnum.dark, SuperThemeEnum.superDark];
 
-/// Theme with enums
-@Tailor(themes: themes)
-class $_ThemeWithEnums {
-  static List<SuperThemeEnum> enums1 = [
-    SuperThemeEnum.light,
-    SuperThemeEnum.superLight,
-    SuperThemeEnum.dark,
-    SuperThemeEnum.superDark
-  ];
-  static List<SuperThemeEnum> enums2 = themesEnums;
-}
+// /// Theme with enums
+// @Tailor(themes: themes)
+// class $_ThemeWithEnums {
+//   static List<SuperThemeEnum> enums1 = [
+//     SuperThemeEnum.light,
+//     SuperThemeEnum.superLight,
+//     SuperThemeEnum.dark,
+//     SuperThemeEnum.superDark
+//   ];
+//   static List<SuperThemeEnum> enums2 = themesEnums;
+// }
```

Comment:
I would remove unused code, if needed you can always get it from git history

Reply:
Uncommented